internal/deviations/deviations.go

```diff
@@ -217,6 +217,8 @@ var (
 	NetworkInstanceTableDeletionRequired = flag.Bool("deviation_network_instance_table_deletion_required", false,
 		"Set to true for device requiring explicit deletion of network-instance table, default is false")
 
+	MissingBgpLastNotificationErrorCode = flag.Bool("deviation_missing_bgp_last_notification_error_code", false, "Set to true to skip check for bgp/neighbors/neighbor/state/messages/received/last-notification-error-code leaf missing case")
```

Comment:
Please use this new format for introducing new deviations: https://github.com/openconfig/featureprofiles/pull/1358.

We will be adding a check for this soon, all new flags should not be exported (define them in lowercase) and accessed from tests through a public accessors (ref in PR 1358)

Reply:
Addressed this.